server/src/main/webapp/WEB-INF/rails/webpack/views/pages/admin_pipelines/admin_pipelines_widget.tsx

```diff
@@ -169,8 +165,24 @@ class PipelineGroupWidget extends MithrilViewComponent<PipelineGroupAttrs> {
 
 export class PipelineGroupsWidget extends MithrilViewComponent<Attrs> {
   view(vnode: m.Vnode<Attrs>) {
+    if (vnode.attrs.sm.hasTarget()) {
+      const target    = vnode.attrs.sm.getTarget();
+      const hasTarget = vnode.attrs.pipelineGroups().some((grp) => grp.name() === target);
+      if (!hasTarget) {
+        const msg = `Either '${target}' pipeline group has not been set up or you are not authorized to view the same.`;
```

Comment:
Provide a learn more link to https://docs.gocd.org/19.12.0/configuration/pipeline_group_admin_config.html ?